doc/content/gcmr/GCMR_Multiphysics_models.md

```diff
@@ -0,0 +1,148 @@
+0.051962, -0.030000, 0.000000
+-0.034641, -0.060000, 0.000000
+-0.000000, -0.060000, 0.000000
+0.034641, -0.060000, 0.000000
+
+
+## SAM Model 
+
+The System Analysis Module (SAM) is a modern system analysis tool being developed by the NEAMS program for advanced non-LWR safety analysis. SAM was chosen as the system-level thermal-hydraulics code due to its capability to model 1D flow-channels at low computational cost. The coolant flow and conduction through the solid structures were modeled through a coupled SAM 1D Fluid – 3D solid simulation with a convective boundary condition at the solid-fluid interface. Temperatures are exchanged at the interface through the MOOSE MultiApps and Transfers systems. The SAM model utilized its built-in Helium material properties, along with 1D fluid components `PBOneDFluidComponent` to simulate the fluid flow of multiple helium coolant channels across the assembly. Additionally, to enable heat transfer between the fluid and solid domains at every time step through the MOOSE transfer system, SAM’s `HeatTransferWithExternalHeatStructure` component along `LayeredAverage` user objects were employed. For steady-state calculations the `SAM_full.i` and `SAM_half.i` are the SAM input files for full and half coolant channels, respectively. 
+
+
+## Multiphysics Coupling 
+
+Multiphysics simulations performed on microreactor models leverage the MOOSE MultiApp system to couple neutronics (Griffin), heat conduction in solid materials (BISON), and coolant channel heat energy transfers (SAM). The coupling logic of these different codes is shown in [Fig_3]. Each code has its own mesh and corresponding space and time scales. 
+
+
+!media media/gcmr/Fig_3.jpg
+      style=display: block;margin-left:auto;margin-right:auto;width:60%;
+      id=Fig_3
+      caption= MultiApp modeling strategy
+
+
+The 3-D power density distribution is calculated by Griffin and transferred to BISON for performing heat conduction calculations to estimate the fuel temperature. The coolant channels are modelled with SAM to estimate the coolant temperature distributions in all coolant channels. The MOOSE Picard fixed point iteration was used to attain a tightly coupled simulation, where the information is transferred back and forth between the three MOOSE-based applications until convergence of the power source distribution. The `MultiApps` blocks in `Griffin_steady_state.i` and `BISON.i` specify the MuliApp coupling for steady-state calculations, and the `MultiApps` blocks in `Griffin_transient.i ` and 
+`BISON_tr.i` specifies the MultiApp coupling for the transient problem. The Transfer system defined in the `Transfers` blocks, takes care of data interchange between the different applications, and controls what type of information is transferred and how the transfer will occur. In this example, the power density and fuel temperatures transferred are defined through the Transfers block in the parent App (Griffin) as follows: 
+
```

Comment:
A little more detail on why choosing those transfer types is needed.

Reply:
It was modified to: The power density and fuel temperatures transferred are defined through the [Transfer system](https://mooseframework.inl.gov/virtual_test_bed/docs/PRs/170/site/resources/multiapps_chps/chp_6_transfers.html) Transfers block in the parent App (Griffin), where [MultiAppProjectionTransfer](https://mooseframework.inl.gov/source/transfers/MultiAppProjectionTransfer.html) was used which performs a projection between the parent App and the child App mesh files while conserving the integral of the transferred data."

With links to the MOOSE documentation on transfer systems.